src/Components/image/Image.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Text from "../../elements/text/Text";
+import styles from "./Image.module.scss";
+import PropTypes from "prop-types";
+
+const Image = ({ image, children, desc }) => {
+  return (
+    <div className={`${styles.background} ${image}`}>
+      <div className={styles.titleWrapper}>
+        <Text heading={"h4"} className={styles.title}>
```

Comment:
Since {children} is a property which can be anything, let's move <Text> and set is a children when <CardImage> component is used. That way instead of pure text, we can pass various blocks of elements in the future. Such as multiple paragraphs of texts and icons.